roles/gateway/README.rst

```diff
@@ -58,6 +61,63 @@ A list of users that don't have their ssh pubkey added to the ``teuthology_user`
     openvpn_users:
       - ovpn: user@host etc...
 
+The following vars are used to populate ``/etc/resolv.conf``.  Defined in the
+secrets repo::
+
+    gw_resolv_search: []
+    # Example: gw_resolv_search: "front.example.com"
+
+    gw_resolv_ns: []
+    # Example:
+    gw_resolv_ns:
+      - 1.2.3.4
+      - 8.8.8.8
+
+The ``gw_networks`` var assumes you have individual "physical" NICs for each
+VLAN in your lab.  The subelements ``peerdns`` and ``dns{1,2}`` are optional.
+``dns1`` and ``dns2`` should be defined under a single NIC and ``peerdns``
+should be set to ``"yes"``.  Defined in the
```

Comment:
Well the role only works on CentOS ;) but I did add verbiage to clear things up: https://github.com/ceph/ceph-cm-ansible/pull/249/commits/fb0bbff518677bd4707659ca06d8f31b8891073a#diff-a52a1791a9e2c8023f940ba001b5ebccR79